modules/ossm-threescale-istio-adapter-troubleshooting.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
+// * service_mesh/v2x/threescale_adapter/threescale-adapter.adoc
+
+[id="ossm-threescale-istio-adapter-troubleshooting_{context}"]
+= 3scale Istio adapter troubleshooting
+
+You might want to check whether the 3scale Istio adapter is working as expected. If this is not the case, you can troubleshoot from the list of scenarios.
+
+Perform the following steps to check that the adapter is working correctly:
+
+. Ensure the _3scale-adapter_ pod is running in the control plane namespace, that is <istio-system>.
+. Check that the _3scale-adapter_ pod has printed out information about itself booting up, such as its version.
+. When performing requests to the services protected by the 3scale adapter integration, always try requests that lack the right credentials and ensure they fail. Check out the 3scale adapter logs to gather additional information.
```

Comment:
Check (vs check out).
Also, point to where they can find the 3scale adapter logs.  Do they have to be enabled?  Can the user set verbosity levels to get more details?

Reply:
@unleashed :point_up: 

Regarding Julie's questions.

Reply:
The logs are standardized in the same place for each pod in the OpenShift console or the `oc` CLI, so nothing special there - it should suffice to reference the documentation that deals with pod logs. I think the log level setting is described in the configuration section, so we could also link to it while mentioning that "you can set a different log level to gather more detailed information". (can't check right now whether the log levels are described in the configuration section, but if not present that's a bug)